src/platforms/esp32/components/avm_builtins/gpio_driver.c

```diff
@@ -475,6 +518,9 @@ static NativeHandlerResult consume_gpio_mailbox(Context *ctx)
     globalcontext_send_message(ctx->global, local_process_id, ret_msg);
     mailbox_remove(&ctx->mailbox);
 
+    if (UNLIKELY(cmd == GPIOCloseCmd)) {
```

Comment:
Why `UNLIKELY`?  (I think by convention we would only use this macro in exigent circumstances)

Reply:
You are right, that may have been a mental lapse, or the fingers were running on autopilot for a minute... ;)

Reply:
I updated this to match the pattern used in `I2C` and `SPI` drivers.